_maps_2024/28-aki.md

```diff
@@ -0,0 +1,15 @@
+---
+day: 28
+creator: Akira Di Sandro
+title: The Great Lakes
+thumbnail: /assets/thumbnails/2024/28-BluePlanet.png
```

Comment:
Looks like your file is named "28-Aki-BluePlanet.png". You'll either need to change the name of the file or update the reference here and below (line 15).
```suggestion
thumbnail: /assets/thumbnails/2024/28-Aki-BluePlanet.png
```